scripts/PewterGym.asm

```diff
@@ -80,20 +81,21 @@ PewterGymScriptReceiveTM34:
 	jp PewterGymResetScripts
 
 PewterGym_TextPointers:
-	dw BrockText
-	dw PewterGymTrainerText1
-	dw PewterGymGuideText
-	dw BeforeReceivedTM34Text
-	dw ReceivedTM34Text
-	dw TM34NoRoomText
+	def_text_pointers
+	dw_const PewterGymBrockText,             TEXT_PEWTERGYM_BROCK
+	dw_const PewterGymCooltrainerMText,      TEXT_PEWTERGYM_COOLTRAINER_M
+	dw_const PewterGymGuideText,             TEXT_PEWTERGYM_GYM_GUIDE
+	dw_const PewterGymBrockWaitTakeThisText, TEXT_PEWTERGYM_BROCK_WAIT_TAKE_THIS
+	dw_const PewterGymReceivedTM34Text,      TEXT_PEWTERGYM_RECEIVED_TM34
+	dw_const PewterGymTM34NoRoomText,        TEXT_PEWTERGYM_TM34_NO_ROOM
 
 PewterGymTrainerHeaders:
 	def_trainers 2
 PewterGymTrainerHeader0:
-	trainer EVENT_BEAT_PEWTER_GYM_TRAINER_0, 5, PewterGymBattleText1, PewterGymEndBattleText1, PewterGymAfterBattleText1
+	trainer EVENT_BEAT_PEWTER_GYM_TRAINER_0, 5, PewterGymCooltrainerMBattleText, PewterGymCooltrainerMEndBattleText, PewterGymCooltrainerMAfterBattleText
```

Comment:
Isn't this guy a Jr Trainer?

Reply:
> Isn't this guy a Jr Trainer?

It's based on the sprite. He uses the CooltrainerM sprite.

Reply:
Hmm. Given that we have things like `FIGHTINGDOJO_HITMONLEE_POKE_BALL`, I'd expect trainers' labels and constants to match their real class. That's already the case for specific characters, e.g. `OPP_ERIKA` has `TEXT_CELADONGYM_ERIKA`, despite using `SPRITE_SILPH_WORKER_F`.

Reply:
All such cases: https://pastebin.com/bj7GFkaE (In at least one case, I think it's the sprite name that should change: `SPRITE_BLUE` should be `SPRITE_RIVAL`, like Gen 2.)